pubsub/google/cloud/pubsub/_http.py

```diff
@@ -203,9 +204,10 @@ def topic_publish(self, topic_path, messages):
         :rtype: list of string
         :returns: list of opaque IDs for published messages.
         """
-        _transform_messages_base64(messages, _base64_unicode)
+        messages_to_send = copy.deepcopy(messages)
```

Comment:
We don't set a hard cap, but we could. I seem to recall the backend having a limit, maybe it's size-based (e.g. 1MB) or number (e.g. 10000 messages) or time (e.g. 60s request).